### PR TITLE
Adding a block for packages that use 'python configure'

### PR DESCRIPTION
**Pull request by Bart-CER**
A new block for software that uses:

python configure
make 
make install

Two new configs depend on this, PyQt and SIP.

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyblocks/generic/configuremakepythonpackage.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    def install_step(self):
+        """Install with cmake install"""
+        return ConfigureMake.install_step(self)
+
+    def sanity_check_step(self, *args, **kwargs):
+        """
+        Custom sanity check for Python packages
+        """
+        #return super(PythonPackage, self).sanity_check_step(EXTS_FILTER_PYTHON_PACKAGES, *args, **kwargs)
```
Comment (boegel) [inline easybuild/easyblocks/generic/configuremakepythonpackage.py]: why is this commented out?

File: easybuild/easyblocks/generic/configuremakepythonpackage.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+@author: Bart Verleye (Centre for eResearch, Auckland)
+"""
+from easybuild.easyblocks.generic.configuremake import ConfigureMake
+from easybuild.easyblocks.generic.pythonpackage import PythonPackage
+from easybuild.tools.filetools import run_cmd
+from easybuild.easyblocks.python import EXTS_FILTER_PYTHON_PACKAGES
+
+
+class ConfigureMakePythonPackage(ConfigureMake, PythonPackage):
+    """Build a Python package and module with cmake.
```
Comment (boegel) [inline easybuild/easyblocks/generic/configuremakepythonpackage.py]: fix docstring, no CMake?

File: easybuild/easyblocks/generic/configuremakepythonpackage.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
+##
+# Copyright 2009-2013 Ghent University
```
Comment (boegel) [inline easybuild/easyblocks/generic/configuremakepythonpackage.py]: fix date

File: easybuild/easyblocks/generic/configuremakepythonpackage.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+EasyBuild support for Python packages that are configured with CMake, implemented as an easyblock
+
+@author: Bart Verleye (Centre for eResearch, Auckland)
+"""
+from easybuild.easyblocks.generic.configuremake import ConfigureMake
+from easybuild.easyblocks.generic.pythonpackage import PythonPackage
+from easybuild.tools.filetools import run_cmd
```
Comment (boegel) [inline easybuild/easyblocks/generic/configuremakepythonpackage.py]: should import from `tools.run` instead, see http://easybuild.readthedocs.org/en/latest/Deprecated-functionality.html#renamed-relocated-functions

**Comment by boegel** [comment]
cleaned up version available in #540

**Comment by Bart-CER** [comment]
Closing this, cleaned up version available in #540.

**Pull request closed without merging**